Signal/src/views/OWSSystemMessageCell.m

```diff
@@ -60,6 +64,12 @@ - (void)commontInit
 
     self.backgroundColor = [UIColor whiteColor];
 
+    self.cellTopLabel = [UILabel new];
+    self.cellTopLabel.textAlignment = NSTextAlignmentCenter;
+    self.cellTopLabel.font = self.topLabelFont;
+    self.cellTopLabel.textColor = [UIColor lightGrayColor];
```

Comment:
Even on our smallest device this takes up ~30% of the view width, so I think we're safe.